opencraft/views.py

```diff
@@ -41,7 +42,13 @@ def get_redirect_url(self, *args, **kwargs):
         Redirect instance manager users to the instance list, and anyone else to the registration form
         """
         user = self.request.user
+        try:
+            beta_test_user = BetaTestApplication.objects.filter(user=user)
+        except TypeError:
+            beta_test_user = []
         if InstanceReference.can_manage(user):
             return reverse('instance:index')
+        elif beta_test_user:
+            return settings.USER_CONSOLE_FRONTEND_URL
```

Comment:
Don't you need to return a `Redirect` here?

Reply:
@giovannicimolin No. It is already a RedirectView, this function simply returns the url to redirect to

Reply:
@giovannicimolin Also, wrong PR :p